docs/conf.py

```diff
@@ -130,18 +127,16 @@
 github_repo = project
 github_user = "DiamondLightSource"
 switcher_json = f"https://{github_user}.github.io/{github_repo}/switcher.json"
-# Don't check switcher if it doesn't exist, but warn in a non-failing way
-check_switcher = requests.get(switcher_json).ok
-if not check_switcher:
-    print(
-        "*** Can't read version switcher, is GitHub pages enabled? \n"
-        "    Once Docs CI job has successfully run once, set the "
-        "Github pages source branch to be 'gh-pages' at:\n"
-        f"    https://github.com/{github_user}/{github_repo}/settings/pages",
-        file=sys.stderr,
-    )
-
-# Theme options for pydata_sphinx_theme
+
+
+# Theme options for pydata_sphinx_theme.
+# We don't check switcher because there are 3 possible states for a repo:
+# 1. New project, docs are not published so there is no switcher
```

Comment:
For case 1, do we still want to catch this and print the suggestion to enable GH pages?

Reply:
How's this?

Reply:
code looks right, made a suggestion on the comment below

Reply:
and probably need to check switcher just in the cli project: https://github.com/DiamondLightSource/python3-pip-skeleton-cli/issues/46